config/main.py

```diff
@@ -101,9 +102,8 @@ def _get_option(ctx,args,incomplete):
         return []
     else:
         breakout_file_input = readJsonFile(BREAKOUT_CFG_FILE)
-        if interface_name in breakout_file_input:
-            breakout_mode_list = [v["breakout_modes"] for i ,v in breakout_file_input.items() if i == interface_name][0]
-            breakout_mode_default = [v["default_brkout_mode"] for i ,v in breakout_file_input.items() if i == interface_name][0]
+        if interface_name in breakout_file_input[INTF_KEY]:
```

Comment:
If possible plz give an example of the contents of the file. It will help for review.